AdyenComponents/BACS Direct Debit/Scenes/Confirmation/BACSConfirmationViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+internal protocol BACSConfirmationViewProtocol: FormViewProtocol {
+    func setUserInteraction(enabled: Bool)
+}
+
+internal class BACSConfirmationViewController: FormViewController, BACSConfirmationViewProtocol {
+
+    // MARK: - Properties
+
+    internal weak var presenter: BACSConfirmationPresenterProtocol?
+
+    // MARK: - Initializers
+
+    internal init(title: String,
+                  styleProvider: FormComponentStyle,
+                  localizationParameters: LocalizationParameters? = nil) {
+        super.init(style: styleProvider)
+        self.title = title
+        self.delegate = self
+        self.localizationParameters = localizationParameters
+    }
+
+    // MARK: - BACSConfirmationViewProtocol
+
+    internal func setUserInteraction(enabled: Bool) {
+        view.isUserInteractionEnabled = enabled
+    }
+}
+
+extension BACSConfirmationViewController: ViewControllerDelegate {
+    internal func viewDidLoad(viewController: UIViewController) {
+        presenter?.viewDidLoad()
```

Comment:
we should send analytics event here, right?

Reply:
Yes, I think we should send it on the first scene right? The input form. What do you think?

Reply:
Done! Take a look at `BACSInputDirectDebitPresenter.swift:54`